src/main/java/frc/robot/Alignable.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+import java.util.function.BiFunction;
+
+public enum Alignable {
+    AMP(0,0,(x,y)-> new Rotation2d(Math.PI / 2)),
```

Comment:
I'm guessing from this that we have the AMP in a fixed angle (90 deg?). Should that not be 270 deg for RED?

Reply:
amp is on the same side for both teams